docs/getting-started/locally/manual.md

```diff
@@ -0,0 +1,28 @@
+# Manual Installation
+
+There are a few steps in a local install of the nio binary that can be done manually instead of using functionality that has been built into the nio Platform for you.
+
+## Download a nio Project Template Using Git
+To download the project template using Git, clone the template and initialize the submodules, which contain the blocks, using the following commands:
+```
+git clone https://github.com/niolabs/project_template.git first_project
+cd first_project
+git submodule update --init --recursive
+```
+
+## Add Blocks Manually
```

Comment:
isn't there a way to do this in your project by cloning blocks in and updating the submodules?

Reply:
We can save that for later—add more manual stuff as it comes up…